server/upstream/upstream-tarballs.txt

```diff
@@ -355,6 +355,7 @@ ibus-unikey:google:ibus-unikey
 iceauth:httpls:http://xorg.freedesktop.org/releases/individual/app/
 ico:httpls:http://xorg.freedesktop.org/releases/individual/app/
 icon-naming-utils:httpls:http://tango.freedesktop.org/releases/
+iio-sensor-proxy:httpls:https://github.com/hadess/iio-sensor-proxy/releases
```

Comment:
Added by you in https://github.com/openSUSE/osc-plugin-collab/commit/1a809b8a :)